### PR TITLE
Explain what we mean by Blazor being "better"

### LINKED ISSUE #14174: Implying that Blazor is better than JavaScript is going to be hard to prove.
While i agree Blazor is AWESOME.  This statement really isn't nice and I think the it will be hard for you to realistically prove that Blazor is **"Better"** than JavaScript.

>normally requires writing JavaScript, but Blazor provides a better approach using C#.

How about rewording it to 

>normally requires writing JavaScript, but Blazor provides another approach using C#.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 36e7dc3c-54c4-35de-203e-d230869e8417
* Version Independent ID: dc5b52d5-c8c6-1440-6b3a-fda898167d0c
* Content: [Build your first Blazor app](https://docs.microsoft.com/en-us/aspnet/core/tutorials/build-your-first-blazor-app?view=aspnetcore-3.0#feedback)
* Content Source: [aspnetcore/tutorials/build-your-first-blazor-app.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/tutorials/build-your-first-blazor-app.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #14174

Let's just say that **one** language is better than **two** languages. We can amend as guided by engineering on review.

Thanks @LindaLawton! :rocket:

### CODE REVIEW

File: aspnetcore/tutorials/build-your-first-blazor-app.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Follow the guidance in the <xref:blazor/get-started> article to create a Blazor
 
 1. Browse to each of the app's three pages in the *Pages* folder: Home, Counter, and Fetch data. These pages are implemented by the Razor component files *Index.razor*, *Counter.razor*, and *FetchData.razor*.
 
-1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing JavaScript, but Blazor provides a better approach using C#.
+1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing code in two languages, JavaScript and C#. Blazor provides a better approach because only one language, C#, is used client- and server-side.
```
Comment (scottaddie) [inline aspnetcore/tutorials/build-your-first-blazor-app.md]: I recommend avoiding the word "better". It's very much a subjective term, and we don't want this to be marketing material.

Comment (guardrex) [reply aspnetcore/tutorials/build-your-first-blazor-app.md]: Dan's word ........ I don't generally change Dan's words on 💀 *pain of death* 💀. :smile: lol

Surely, it says the same thing if it's just ...

> Blazor provides an approach that only requires one language, C#.

... and we'll let the reader decide if that's "better." *Stand-by!*

Comment (guardrex) [reply aspnetcore/tutorials/build-your-first-blazor-app.md]: **_Done!_**

File: aspnetcore/tutorials/build-your-first-blazor-app.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Follow the guidance in the <xref:blazor/get-started> article to create a Blazor
 
 1. Browse to each of the app's three pages in the *Pages* folder: Home, Counter, and Fetch data. These pages are implemented by the Razor component files *Index.razor*, *Counter.razor*, and *FetchData.razor*.
 
-1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing JavaScript, but Blazor provides a better approach using C#.
+1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage normally requires writing code in two languages, JavaScript and C#. Blazor provides an approach that only requires one language, C#.
```
Comment (danroth27) [inline aspnetcore/tutorials/build-your-first-blazor-app.md]: How about this?

```suggestion
1. On the Counter page, select the **Click me** button to increment the counter without a page refresh. Incrementing a counter in a webpage would normally require writing some JavaScript, but with Blazor you can instead write C#.
```

Comment (guardrex) [reply aspnetcore/tutorials/build-your-first-blazor-app.md]: No future tense tho.
"Some" is very conversational and not specific.

Stand-by ... I'll make minor edits to the suggestion for another look.

File: aspnetcore/tutorials/build-your-first-blazor-app.md (hunk 1 above)
Comment (danroth27) [reply aspnetcore/tutorials/build-your-first-blazor-app.md]: I have many opinions, but I don't guarantee that any of them are right 😃. I agree that removing "better" here makes sense.

File: aspnetcore/tutorials/build-your-first-blazor-app.md (hunk 2 above)
Comment (guardrex) [reply aspnetcore/tutorials/build-your-first-blazor-app.md]: @danroth27 Try that updated suggestion :point_up:

Comment (danroth27) [reply aspnetcore/tutorials/build-your-first-blazor-app.md]: Looks good to me!
